packages/cli/src/sso.ee/saml/saml.service.ee.ts

```diff
@@ -135,6 +135,11 @@ export class SamlService {
 			});
 		}
 
+		const binding = this.identityProviderInstance.entityMeta.getSingleSignOnService('redirect');
```

Comment:
since saml service has a property SamlValidator, would it make sense for the validator to expose a method to validate binding that could be called from both file ?